datalad/interface/utils.py

```diff
@@ -514,16 +515,24 @@ def default_result_renderer(res):
                 if res.get('message', None) else ''))
 
 
-def _display_suppressed_message(nsimilar, ndisplayed, final=False):
+def _display_suppressed_message(nsimilar, ndisplayed, last_ts, final=False):
```

Comment:
This direction sounds good to me, but I guess that the rate-limiting shouldn't be in effect when `final=True`?

Reply:
Good point! Adjusted, thx.